docs/reference-guide/exposing-system-metrics.md

```diff
@@ -0,0 +1,257 @@
+---
+id: exposing-system-metrics
+title: Exposing system metrics (Enterprise)
+sidebar_label: Exposing system metrics
+---
+
+In production systems, monitoring of applications is crucial, and that includes databases as well. Since Memgraph v2.8,
+an HTTP server has been added in order to expose system metrics, giving developers ability to track information about transactions,
+query latencies, snapshot recovery latencies, triggers, bolt messages, indexes, streams, and many more.
+
+## Configuring the HTTP endpoint
+
+Exposing metrics is a Memgraph Enterprise feature and therefore needs a valid Memgraph Enterprise license key.
+After successfully entering the license key, Memgraph would need to be restarted in order to start the metrics HTTP
+server in the following run.
+
+The default address and port for the metrics server is `0.0.0.0:9091`, and can be configured with configuration flags
+`--metrics-address` and `--metrics-port`. For more info on the configuration, check the [list of the configuration variables](/reference-guide/configuration.md).
+
+
+## Types of system metrics
+
+In Memgraph, there are 3 types of metrics that measure different parts of the system in order to ensure proper
+monitoring has been done. Those 3 metrics are:
+- **Gauge** - a single value of some variable in the system (e.g. memory usage)
+- **Counter (uint64_t)** - a value that can be incremented or decremented (e.g. number of active transactions in the system)
+- **Histogram (uint64_t)** - distribution of measured values (e.g. certain percentile of query latency on N measured queries)
+
+
```

Comment:
```suggestion
```

Reply:
why do I need to delete everything?

Reply:
I've repositioned it, but now i don't know where that suggestion disappeared!